config/application.rb

```diff
@@ -60,8 +60,6 @@ class Application < Rails::Application
       :'zh-TW',
     ]
 
-    config.i18n.default_locale = :en
```

Comment:
No, I'm afraid you cannot do this. This option "default_locale" is not the same as pre-choosing a locale to be displayed. It controls the fallback direction for missing translations. When a translation is missing, it gets the string from the english one instead. This makes sense because the software is developed in english so that locale always contains all the keys. But if another is chosen here, things will break and error out because keys will be genuinely missing without a fallback.